docs/tutorial/advanced/mkda_association.md

```diff
@@ -0,0 +1,92 @@
+
+## MKDA Chi-Squared
+
+In our example we want to know if and where studies of nicotine administration show more consistent brain activiation, than *all other studies* in the Neurosynth database (15,000+ studies). 
+
+We can perform this test using the `Multilevel kernel density (MKDA) analysis- Chi-square` analysis, originally introduced in [Wager et al.,](https://doi.org/10.1093/scan/nsm015). For every voxel, can then test if a greater proportion of studies in our meta-analysis consistently activate a given voxel greater than what we observe in a large set of studies that we did not select for our meta-analysis. 
+
+Conceptually, this tests if there's evidence of a *population level* association between the task or psychological construct in our meta-analysis and brain activation (for every voxel).
+
+:::info   
+**What happened to the "forward inference" and "reverse inference" analysis?**
+
+On Neurosynth.org, we renamed the pre-generated forward and reverse inference maps; they're now referred to as the "uniformity test" and "association test" maps, performed by the MKDA Chi-Squared algorithm.
+
+Although the method we used hasn't change,d the latter names more accurately capture what these maps actually mean. It was a mistake on our part to have used the forward and reverse inference labels; those labels should properly be reserved for probabilistic maps generated via a Bayesian estimation analysis, rather than for z-scores resulting from frequentist inferential tests. These maps are more difficult to interpret and use correctly, which is why we don't currently support this approach.
+
+See "Further Reading" below to read more about why this change was made
+:::
+
+## Specifying MKDA Chi-Squared Meta-Analysis
+
+Specifying an MKDA Chi-Square meta-analysis in Neurosynth is easy. Simply, select a target set of studies as you would for any other meta-analysis, using either automated or manual selection methods. 
+
+In Step 3 ("Create Meta-Analysis Specification") of your Project, select *MKDAChi2* as the *algorithm*. 
+
+
+![MKDA Chi Squared](/tutorial/mkda_chi_squared_algo.png). 
+
+Next, select the annotation inclusion column you want to use, as before (by default, the "included" column will be used).
+
+Now, select a reference dataset from the dropdown list below. The Neurosynth dataset represents the latest release of the legacy *Neurosynth* dataset (version 7), released July, 2018. The *Neurostore* dataset represents the latest update of our continuously updating "live" dataset, spanning over 20,000 neuroimaging studies. 
```

Comment:
It's probably worth noting that there are issues with comparing a manually-curated sample against an automated reference dataset, such as sampling biases in either and coordinate extraction. I don't think that merits more than a small admonition box.

Another concern is that you'll want to exclude any relevant studies from the reference dataset, so users should probably make at least some effort to exclude studies investigating the constructs or tasks they are studying with their meta-analysis.

Reply:
Also, as you mentioned on Slack, you could add in a small admonition saying that users could employ this estimator to do a two-sample comparison like the ALE Contrast analysis.

Reply:
This is a good point but I'm not what the answer is.

The workflow will exclude studies from the reference sample that you have in your meta-analysis. 

I think the main issue is if you don't include all relevant studies are just a subset, right?